src/schema/users.ts

```diff
@@ -846,7 +846,7 @@ export const resolvers: IResolvers<any, Context> = {
         .getRawMany();
     },
     userStreak: async (_, __, ctx: Context, info): Promise<GQLUserStreak> => {
-      const streak = await graphorm.queryOneOrFail<GQLUserStreakTz>(
+      const streak = await graphorm.query<GQLUserStreakTz>(
```

Comment:
Can still use queryOne right? Withouth the fail part

Reply:
Don't think there is a queryOne, at least my autocomplete didn't suggest it.

Reply:
We can add one, its part of graphorm, but this is fine as well yes.

Reply:
We can add one in graphorm, it would be the same as `queryOneOrFail` just without `not found` throw. Non blocking yes.

Reply:
I'll open a new PR for that. Do you think we should return null when nothing is found? Because `res[0]` itself might cause an issue if nothing is found.

Reply:
Opened #1728